app/validators/url-field.ts

```diff
@@ -0,0 +1,49 @@
+import buildMessage from 'ember-changeset-validations/utils/validation-errors';
+import { validateFormat } from 'ember-changeset-validations/validators';
+
+const Protocols = {
+    http: 'http:',
+    https: 'https:',
+};
+
+interface Options {
+    acceptedProtocols?: string[];
+    translationArgs?: {
+        description: string,
+    };
+}
+
+export function validateHttpUrl(options?: Options) {
```

Comment:
What if we called it something more like `validateUrlWithProtocols` since it technically doesn't need to be an `http` url?